server/src/metadata/commands/run-tenant-migrations.command.ts

```diff
@@ -0,0 +1,45 @@
+import { Command, CommandRunner, Option } from 'nest-commander';
+
+import { TenantMigrationService } from 'src/metadata/tenant-migration/tenant-migration.service';
+import { MigrationRunnerService } from 'src/metadata/migration-runner/migration-runner.service';
+
+// TODO: implement dry-run
+interface RunTenantMigrationsOptions {
+  workspaceId: string;
+}
+
+@Command({
+  name: 'tenant:migrate',
+  description: 'Run tenant migrations',
+})
+export class RunTenantMigrations extends CommandRunner {
```

Comment:
@Weiko The mismatch between the CommandRunner class and the CommandModule is a bit strange, we should align them I guess:
`TenantMigrationCommandModule`
`TenantMigrationCommandRunner`

Reply:
@magrinj I'm not sure I understand.
RunTenantMigrations and SyncTenantMetadata classes are both command classes extending CommandRunner and are both provided by the same module MetadataCommandModule. Do you mean I should have 1 module per command?